### PR TITLE
RepoSense Config

### PR DESCRIPTION
**Pull request by speezy37**
This will solve faceless problem.

Please check your name, GitHub ID and author name.
Author name is the name used for previous commits that are not your GitHubID.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SetScheduleCommand.java [hunk 1]
```diff
@@ -72,6 +65,14 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
         }
 
         Person personToEdit = lastShownList.get(index.getZeroBased());
+        /**
```
Comment (jinyao-lee) [inline src/main/java/seedu/address/logic/commands/SetScheduleCommand.java:68]: Might want to edit the javadoc comments as the conditions have changed

File: _reposense/config.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	},
+	{
+	  "githubId": "jylee-git",
+      "displayName": "Lee Jin Yao",
+      "authorNames": ["Jin Yao", "Lee Jin Yao"]
+	},
+	{
+	  "githubId": "Woonhian",
+      "displayName": "Lim Woon Hian",
+      "authorNames": ["Woonhian", "Lim Woon Hian"]
```
Comment (Woonhian) [inline _reposense/config.json:22]: Yup correct, very good!

**Review by Woonhian — APPROVED** [review]
Very good!

**Review by pinjuen — APPROVED** [review]
very good!

**Review by Hafizuddin-github — APPROVED** [review]
Very good!

**Review by jinyao-lee — APPROVED** [review]
PR approved. Good job in configuring the RepoSense for authorship tracking

**Pull request merged**